### PR TITLE
Add ITRocket endpoints

### PR DESCRIPTION
**Pull request by itrocket-team**
## Summary by CodeRabbit

- **New Features**
  - Introduced new endpoints for Mainnet Beta:
    - RPC: `celestia-mainnet-rpc.itrocket.net`
    - API: `celestia-mainnet-api.itrocket.net`
    - gRPC: `celestia-mainnet-grpc.itrocket.net:443`
    - Consensus: `celestia-mainnet-consensus.itrocket.net`
    - Explorer: [https://mainnet.itrocket.net/celestia/](https://mainnet.itrocket.net/celestia/)

### CODE REVIEW

File: nodes/mainnet.md [hunk 1]
```diff
@@ -126,6 +126,7 @@ to participate in Mainnet Beta:
 - `rpc-celestia.staker.space`
 - `celestia-rpc.noders.services`
 - `celestia.moonli.me`
+- `celestia-mainnet-rpc.itrocket.net`
```
Comment (jcstein) [inline nodes/mainnet.md:129]: what ports are open?

File: nodes/mainnet.md [hunk 2]
```diff
@@ -153,6 +154,7 @@ to participate in Mainnet Beta:
 - `api-celestia.staker.space`
 - `celestia-api.noders.services`
 - `celestia.moonli.me/api`
+- `celestia-mainnet-api.itrocket.net`
```
Comment (jcstein) [inline nodes/mainnet.md:157]: what ports are open?

Comment (itrocket-team) [reply nodes/mainnet.md:157]: 443

File: nodes/mainnet.md (hunk 1 above)
Comment (itrocket-team) [reply nodes/mainnet.md:129]: 443

Comment (itrocket-team) [reply nodes/mainnet.md:129]: Archive node, discard_abci_responses = false

File: nodes/mainnet.md (hunk 2 above)
Comment (jcstein) [reply nodes/mainnet.md:157]: could you add this in bullets underneath please?

File: nodes/mainnet.md (hunk 1 above)
Comment (jcstein) [reply nodes/mainnet.md:129]: same here: could you add this in bullets underneath please?

**Pull request closed without merging**